### PR TITLE
Add support for instagram login

### PR DESCRIPTION
**Pull request by hlbraddock**

### CODE REVIEW

File: app/views/modals/loginDialog.html [hunk 1]
```diff
@@ -12,6 +12,10 @@ <h3 ng-if="status401" translate>Authorization Required</h3>
       Please sign in with one of the following services:
     </p>
     <div class="row">
+      <div ng-if="instagramLogin" class="{{relayLogin ? 'col-sm-5 col-sm-offset-1' : 'col-sm-6 col-sm-offset-3' }}">
```
Comment (hlbraddock) [inline app/views/modals/loginDialog.html]: This change alone is not sufficient. Do you have the expertise to quickly and easily modify it? Or is it just something I should tinker with until I get it right? @OzzieOrca

Comment (OzzieOrca) [reply app/views/modals/loginDialog.html]: You have to adjust the size of the other columns. I could play with it locally if you aren't getting anywhere. Bootstrap has a grid size of 12 so everything in the row has to add up to 12.

Comment (hlbraddock) [reply app/views/modals/loginDialog.html]: I've updated this commit and this PR is ready for review. Thanks! @OzzieOrca

File: languages/ert.pot [hunk 2]
```diff
@@ -509,10 +509,14 @@ msgid "Create New Event"
 msgstr ""
```
Comment (hlbraddock) [inline languages/ert.pot]: Is `languages/ert.pot` meant to be auto generated? @OzzieOrca

Comment (OzzieOrca) [reply languages/ert.pot]: I think running `yarn angular-gettext-extract` generates it. https://github.com/CruGlobal/conf-registration-web/blob/master/package.json#L76

Do we have any live translations? I don't see a whole lot of tooling for translations. I've added OneSky upload/download scripts for MissionHub web and mobile that run on Travis deploys.

File: app/scripts/app.config.js [hunk 3]
```diff
@@ -30,15 +30,15 @@ angular.module('confRegistrationWebApp')
       },
       vars: {
         development: {
-          apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
```
Comment (OzzieOrca) [inline app/scripts/app.config.js]: Did you mean to commit this?

File: languages/ert.pot (hunk 2 above)
Comment (hlbraddock) [reply languages/ert.pot]: ERT isn't officially internationalized, so I don't know why we're doing this anyway.

File: app/scripts/app.config.js [hunk 4]
```diff
@@ -30,15 +30,15 @@ angular.module('confRegistrationWebApp')
...
           tsysEnvironment: 'staging'
         },
         staging: {
-          apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
           tsysEnvironment: 'production'
         },
         production: {
-          apiUrl: 'https://api.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
```
Comment (OzzieOrca) [inline app/scripts/app.config.js]: We can't go to production with this. Changing the `development` one should have been enough if you were testing on localhost. This environment library just does string matching with the domain in the address bar.

**Comment by hlbraddock** [comment]
https://github.com/CruGlobal/conf-registration-web/blob/instagram-login/app/img/signin-instagram.jpeg and https://github.com/CruGlobal/conf-registration-web/blob/instagram-login/app/img/signin-instagram-grey.jpeg are actually identical images. I didn't grey one out. And I don't know where it's used.

**Comment by OzzieOrca** [comment]
Was that a question? If you added the grey version and aren't using it, let's delete it. There may have been some old unused versions (maybe the grey ones) of the other sign in logos running around.

File: app/scripts/app.config.js (hunk 3 above)
Comment (hlbraddock) [reply app/scripts/app.config.js]: My mistake. I keep inadvertently including that in the commit. Resolved.

**Pull request merged**